dicom-archive/dicomSummary.pl

```diff
@@ -4,6 +4,55 @@
 # Perl tool based on DCMSUM.pm and DICOM.pm to create a summary report for a given dir containing dicoms
 # @VERSION : $Id: dicomSummary.pl 4 2007-12-11 20:21:51Z jharlap $
 
+=pod
+
+=head1 NAME
+
+dicomSummary.pl -- prints out an informative summary for DICOMs in a given directory
+
+=head1 SYNOPSIS
+
+perl dicomSummary.pl </PATH/TO/DICOM/DIR> [ -compare </PATH/TO/DICOM/COMPARE/DIR> ] [ -tmp </PATH/TO/TMP/DIR> ] `[options]`
+
+Available options are:
+
+-comparedir: path to another DICOM directory to compare with
+
+-dbcompare : run a comparison with entries int he database
+
+-database  : use the database
+
+-dbreplace : use this option only if the DICOM data changed and need to be updated
+             in the database
+
+-profile   : specify the name of the config file residing in C<.loris_mri> of the
+             current directory
+
+-tmp       : to specify a temporary directory. It will contain the summaries if
+             used with -noscreen option
+
+-xdiff     : to see with tkdiff the result of the two folders comparison or the
+             comparison with the database content with
+
+-batch     : run in batch mode if set. Will log differences to a /tmp/diff.log file.
+
+-verbose   : be verbose if set
+
+-version   : print CVS version number and exit
+
+
+=head1 DESCRIPTION
+
+A tool for producing an informative summary for DICOMs in a given directory
+(scanner information, acquisitions list, acquisitions parameters...). This tool
+can also compare the DICOM data present in two directories or compare the DICOM
+data present in a given directory and what is stored in the database.
```

Comment:
...givern directory **with** what is stored in the database.

Reply:
done